app/views/devise/sessions/_login.html.erb

```diff
@@ -35,7 +35,7 @@
   <div class="row">
     <div class="col-md-12">
       <h1 class="text-center">
-        Connect to MyUSA
```

Comment:
The change to "Connect with" from "Connect to" is the main change drawing specifically on user research, not just copy editing best practices. See my comment in the conversation for more.